packages/react/src/Field/Field.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { render } from '@testing-library/react'
+import { createRef } from 'react'
+import { Field } from './Field'
+import '@testing-library/jest-dom'
+
+describe('Field', () => {
+  it('renders', () => {
+    const { container } = render(<Field />)
+    const component = container.querySelector(':only-child')
+
+    expect(component).toBeInTheDocument()
+    expect(component).toBeVisible()
+  })
+
+  it('renders a design system BEM class name', () => {
```

Comment:
Why do we care about the particular CSS framework or strategy? Isn't this sort of an implementation detail? What if we want to use something else than BEM at a later point and we refactor that? Is it not better to use some CSS linter that is aware of BEM?

Reply:
Class names are the essential API of our HTML components. They are less visible for now, but consider each React component an implementation of the bare HTML that it renders. The class names we choose as well as their syntax inform the markup that the JavaScript Libraries on top of our CSS Library must implement. That’s why React components must test that they render the expected class names.